Classes/Console/Service/XsdGenerator.php

```diff
@@ -257,7 +258,7 @@ protected function getClassNamesInNamespace($namespace)
         $packageKey = $this->getPackageKeyFromNamespace($namespace);
         $viewHelperClassFilePaths[] = $this->packageManager->getPackage($packageKey)->getPackagePath() . 'Classes/ViewHelpers/';
         if ($packageKey === 'fluid') {
-            $viewHelperClassFilePaths[] = realpath(PATH_site . 'typo3/') . '/../vendor/typo3fluid/fluid/src/ViewHelpers/';
+            $viewHelperClassFilePaths[] = Environment::getProjectPath() . '/vendor/typo3fluid/fluid/src/ViewHelpers/';
```

Comment:
as both, vendor path and project path are configurable, this won't work in case they deviate from the default.

Therefore I suggest to use class [reflection of a fluid class](https://www.php.net/manual/de/reflectionclass.getfilename.php#121912) and from that determine the vendor directory. While this still can change, when fluid changes the class/file structure, I would consider it stable enough for minor releases.